.github/workflows/deploy-staging.yml

```
@@ -89,6 +97,42 @@ jobs:
+      SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
+      DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+      DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+
+  infra-hl7-notification-routing-stack-staging:
+    if: needs.files-changed.outputs.infra-hl7-notification-routing-stack == 'true'
+    uses: ./.github/workflows/_deploy-hl7-notification-routing-cdk.yml
+    needs: files-changed
+    with:
+      deploy_env: "staging"
+      secrets_cdk_stack: ${{ vars.SECRET_STACK_NAME_STAGING }}
+      cdk_stack: ${{ vars.HL7_NOTIFICATION_ROUTING_STACK_NAME_STAGING }}
```

actionlint (1.7.4)

input "cdk_stack" is not defined in "./.github/workflows/_deploy-hl7-notification-routing-cdk.yml" reusable workflow. defined inputs are "AWS_REGION", "deploy_env", "hl7_notification_routing_stack", "secrets_cdk_stack"

(workflow-call)